examples/aa-simple-dapp/src/hooks/useAlchemyProvider.ts

```diff
@@ -35,7 +35,7 @@ export const useAlchemyProvider = () => {
         .connect((provider) => {
           return createMultiOwnerMSCA({
             rpcClient: provider,
-            signer,
+            owner: signer,
```

Comment:
interesting, did this work before? wouldn't signer need to have been under the owner field anyways?

Reply:
I broke this in a previous PR and fixed it recently, but forgot to change here